lib/pinchflat_web/controllers/pages/page_html/history_table_live.ex

```diff
@@ -56,12 +56,13 @@ defmodule Pinchflat.Pages.HistoryTableLive do
     """
   end
 
-  def mount(_params, _session, socket) do
+  def mount(_params, session, socket) do
     page = 1
-    base_query = generate_base_query()
+    media_state = session["media_state"]
+    base_query = generate_base_query(media_state)
     pagination_attrs = fetch_pagination_attributes(base_query, page)
 
-    {:ok, assign(socket, Map.merge(pagination_attrs, %{base_query: base_query}))}
+    {:ok, assign(socket, Map.merge(pagination_attrs, %{base_query: base_query, media_state: media_state}))}
```

Comment:
nit: you don't need to pass `media_state` in the new assigns since we're not currently doing anything with that outside of the `mount` function.

Since that's the case, I figure you can also save the intermediate variable and just use:

```elixir
base_query = generate_base_query(session["media_state"])
# ...
{:ok, assign(socket, Map.merge(pagination_attrs, %{base_query: base_query}))}
```